nemo/collections/multimodal/speech_llm/data/build_dataset.py

```diff
@@ -147,16 +133,25 @@ def build_speechllm_dataloader(dataset, data_cfg, consumed_samples=0, is_predict
                     assert len(input_cfg) == 1, "Only one dataset with multiple manifest paths is supported for eval"
                     data_cfg.input_cfg = input_cfg
                     # for getting names
-                    manifest_filepath = [ic.manifest_filepath for ic in input_cfg[0].input_cfg]
+                    manifest_filepath = []
+                    for ic in input_cfg[0].input_cfg:
+                        if hasattr(ic, "manifest_filepath"):
+                            manifest_filepath.append(ic.manifest_filepath)
+                        else:
+                            assert ic.type == "txt_pair"
+                            manifest_filepath.append(ic.target_paths)
```

Comment:
Is there a reason why we have to use different attribute for `txt_pair`? It looks better if it has the same `manifest_filepath` input as other types

Reply:
@pzelasko could you please clarify this?

Reply:
yes, followed up offline - TL;DR this is to support an NMT format where source and target translations are stored in separate plain text files